assets/js/src/settings/pages/signup_confirmation/index.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import EnableSignupConfirmation from './enable_signup_confirmation';
+
+export default function SignupConfirmation() {
```

Comment:
As far as I know, we don't use the name `index` for files.  We always use the folder name. See `form` or `listing` for example.

Reply:
I use `index.tsx` so that I can use the folder name in the import
```ts
import SignupConfirmation from 'settings/page/signup_confirmation';
```
This is a convention used widely by many libraries.

Reply:
I understand that @amine-mp but that convention has not been used by MailPoet.